webpack/assets/javascripts/react_app/redux/reducers/hosts/index.js

```diff
@@ -1,8 +1,6 @@
 import { combineReducers } from 'redux';
 import storage from './storage';
-import powerStatus from './powerStatus';
 
 export default combineReducers({
```

Comment:
Is combineReducers still needed?

Reply:
yes, to keep on the original store structure, see:
![Screenshot from 2019-12-12 12-19-54](https://user-images.githubusercontent.com/26363699/70703920-b3e79100-1cd9-11ea-8952-11c0952b465b.png)